sample_code/ep101/README.md

```diff
@@ -0,0 +1,26 @@
+# [namedtuple to json and other pitfalls (intermediate)](https://youtu.be/QOKqvuvA3ok)
+
+Today we continue talking about namedtuples -- today how to convert them to json and the pitfalls of being a tuple.
+
+## Interactive examples
+
+### Python
+
+```python
+import json
+user = User(name='Anthony', age=29, location='CA')
+json.dumps(user)
+print(json.dumps(user))
+print(json.dumps(('Anthony', 29, 'CA')))
+user._asdict()
+print(json.dumps(user._asdict(), indent=2))
+user._fields
+user._field_types  # user.__annotations__
```

Comment:
It seems that `user._field_types` is [deprecated in 3.8 and removed in 3.9](https://docs.python.org/3/library/typing.html?highlight=namedtuple#typing.NamedTuple).